src/include/pmix_atomic.h

```diff
@@ -112,12 +127,12 @@ static inline void pmix_atomic_rmb(void)
         return __atomic_##name##_fetch(addr, value, __ATOMIC_RELAXED);         \
     }
 
+#endif
+
```

Comment:
In theory we are missing a branch here. Maybe force fail if neither conditions were met?